index.js

```diff
@@ -11,16 +11,39 @@ const nextButton = document.querySelector('[data-js="button-next"]');
 const pagination = document.querySelector('[data-js="pagination"]');
 
 // States
-const maxPage = 1;
-const page = 1;
+let maxPage = "";
```

Comment:
I recommend that you set a number as a default value as later on this is going to be a number as well.
```suggestion
let maxPage = 1;
```